graql/reasoner/recursion.feature

```diff
@@ -1633,6 +1633,8 @@ Feature: Recursion Resolution
 
 
   # TODO: re-enable all steps when resolvable (currently takes too long) (#75)
+  #  TODO: disabled because down and side casting no longer allowed
+  @ignore
```

Comment:
I don't believe this scenario does any downcasting or sidecasting.

Reply:
It does. rule-3 has `$y isa S;` as a conclusion

Reply:
Yep true, missed that one